.eslintrc.json

```diff
@@ -101,6 +99,48 @@
           }
         ]
       }
+    },
+    // rules which apply only to Markdown
+    {
+      "files": [
+        "**/*.{md,mdx}"
+      ],
+      "extends": [
+        "plugin:mdx/recommended"
+      ],
+      "settings": {
+        "mdx/code-blocks": true
+      }
+    },
+    {
+      "files": [
+        "**/*.{md,mdx}",
+        "**/*.{md,mdx}/**"
+      ],
+      "parserOptions": {
+        "project": false
```

Comment:
To unset the project config of tslint parser was needed to disable advanced language services, as they're not available on a markdown execution context